### PR TITLE
gradient animation added

### PR DESCRIPTION
**Pull request by NataliaKiwibird**
Contribution:
- added new file gradient.css with new animation
- added class="gradient" to index.html
- animations.js updated

This request is made as a contribution to Hacktoberfet 2020.

### CODE REVIEW

File: animations/gradient.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  background-color: navy;
+  animation-name: gradient;
+  animation-duration: 3s;
+  animation-iteration-count: infinite;
+}
+h1{
+color: white;
+font-size: 5em;
+text-align: center;
+padding-top: .4em;
```
Comment (NiallEccles) [inline animations/gradient.css:18]: Could you move these styles to be within your `.gradient` class?

File: index.html [hunk 2]
```diff
@@ -29,9 +29,11 @@
...
 		<section class="section text-center">
 			<!--  here is the sample animation -->
+			<div class="gradient">
 			<h1 id="target" class="title scare" data-text="Hacktoberfest animations">
 				Hacktoberfest animations
 			</h1>
+			</div>
 			<!--  here the animations will be inserted -->
 			<select id="animationSelect"></select>
 			<!--  here we show the animation name and who contribute with it -->
```
Comment (NiallEccles) [inline index.html:39]: Could you undo your changes to this file? The JavaScript will take care of applying your class when your animation is selected.

**Review by NiallEccles — CHANGES_REQUESTED** [review]
Hey there! Thanks for submitting this pull request. I just left two small changes for you to make before I can merge this.

File: animations/gradient.css (hunk 1 above)
Comment (NataliaKiwibird) [reply animations/gradient.css]: Hi, the changes are done. Thank you

Comment (NataliaKiwibird) [reply animations/gradient.css]: Hi, the changes are done. Thank you

**Pull request merged**